utils/i18n.go

```diff
@@ -36,7 +36,10 @@ func InitTranslations() {
 func GetTranslationsBySystemLocale() i18n.TranslateFunc {
 	locale := model.DEFAULT_LOCALE
 	if userLanguage, err := jibber_jabber.DetectLanguage(); err == nil {
-		locale = userLanguage
+		// only set the system locale if is supported, fallback to DEFAULT_LOCALE
```

Comment:
It wasn't working I tried setting environment var LANG=fr and it didn't fallback to DEFAULT_LOCALE it just stated the the system translations failed to load, maybe if we change that to panic?